src/current/cockroachcloud/cluster-management.md

```diff
@@ -138,16 +133,15 @@ The ability to remove a region from a cluster through the Console is temporarily
 
     If you have a GCP cluster with [VPC peering]({% link cockroachcloud/network-authorization.md %}) enabled, the IP range will be automatically populated for added regions.
 
-1. From the **Choose a region** dropdown, select the region you want to use.
-1. From the **Nodes** dropdown, select the number of nodes in the new region.
+1. In the Regions & Nodes section, select the desired new region and specify the number of nodes for it.
 1. In the **Summary** sidebar, verify the hourly estimated cost for the cluster.
-1. Click **Continue to payment**.
-1. In the **Confirmation** dialog, verify your new cluster configuration.
-1. Click **OK**.
+1. Click **Update**.
 
 ### Remove a region from your cluster
 
-When you remove a region from a [multi-region]({% link cockroachcloud/plan-your-cluster.md %}#multi-region-clusters) cluster, the node in that region with the highest ordinal will be [decommissioned](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/node-shutdown?filters=decommission#decommission-the-node) first. Any ranges on that node will be [up-replicated](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/ui-replication-dashboard#snapshot-data-received) to other nodes, and once decommission is complete that node will be shut down. This process is then repeated for every other node in the region. To remove a region from your cluster:
+When you remove a region from a [multi-region]({% link cockroachcloud/plan-your-cluster.md %}#multi-region-clusters) cluster, the node in that region with the highest ordinal will be [decommissioned](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/node-shutdown?filters=decommission#decommission-the-node) first. Any ranges on that node will be [up-replicated](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/ui-replication-dashboard#snapshot-data-received) to other nodes, and once decommission is complete that node will be shut down. This process is then repeated for every other node in the region. 
```

Comment:
I'm curious what happens if you have say 3 regions and you need to go down to 1 region (2 regions isn't allowed). Do you need to remove both of the excess regions at the same time, and do we tear them down in parallel so that you end up finally with just one?

Reply:
Yep, 2 regions are deleted at once, so you end up with only one region.

Reply:
@mdlinville Added the note!